br/backup-storage-S3.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Back Up and Restore Data on Amazon S3 Using BR
+summary: Learn how to use BR to back up and restore data to Amazon S3 storage.
+---
+
+# Back Up and Restore Data on Amazon S3 Using BR
+
+The Backup & Restore (BR) tool supports using Amazon S3 or other Amazon S3-compatible file storages as the external storage for backing up and restoring data.
+
+## Application scenarios
+
+By using Amazon S3, you can quickly back up the data of a TiDB cluster deployed on Amazon EC2 to Amazon S3, or quickly a TiDB cluster from the backup data in Amazon S3.
+
+## Configure privileges to access S3
+
+Before backup or restoration using S3, you need to configure the privileges to access S3.
+
+### Configure access to the S3 directory
+
+Before backup, configure the following privileges to access the backup directory of S3.
+
+- Minimum privileges for TiKV and BR during backup, able to access backup directories `s3:ListBucket`, `s3:PutObject`, and `s3:AbortMultipartUpload`
+- Minimum privileges for TiKV and BR during restoration, able to access backup directories `s3:ListBucket` and `s3:GetObject`
+
+If you have not yet created a backup directory, refer to [AWS Official Document](https://docs.aws.amazon.com/AmazonS3/latest/userguide/create-bucket-overview.html) to create an S3 bucket in the specified Region. If necessary, you can also create a folder in the bucket by referring to [AWS official documentation](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-folders.html).
+
+### Configure a user to access S3
+
+It is recommended that you configure access to S3 using either of the following ways:
+
+- Associate an IAM role that can access S3 with the EC2 instances where the TiKV and BR nodes run. After the association, BR can access the backup directories of S3.
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    br backup full --pd "${PDIP}:2379" --storage "s3://${Bucket}/${Folder}" --s3.region "${region}"
+    ```
+
+- Before running the `br backup` command, configure `access-key` and `secret-access-key` to access S3, and set `-send-credentials-to-tikv=true` to pass the access key from BR to each TiKV.
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    br backup full --pd "${PDIP}:2379" --storage "s3://${Bucket}/${Folder}?access-key=${accessKey}&secret-access-key=${secretAccessKey}" --s3.region "${region}" --send-credentials-to-tikv=true
+    ```
+
+Considering that the access key in a command is vulnerable to leakage, you are advised to associate IAM role to EC2 instances to access S3.
```

Comment:
```suggestion
Because the access key in a command is vulnerable to leakage, you are recommended to associate an IAM role to EC2 instances to access S3.
```